coffee-shop/src/store/addressSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const addressSlice = createSlice({
+  name: "address",
+  initialState: {
+    address: "",
```

Comment:
I think address is object of 2 keys : title, details

Reply:
Done